options/options_helper.cc

```diff
@@ -1901,10 +1901,6 @@ std::unordered_map<std::string, OptionTypeInfo>
          {offset_of(&CompactionOptionsFIFO::max_table_files_size),
           OptionType::kUInt64T, OptionVerificationType::kNormal, true,
           offsetof(struct CompactionOptionsFIFO, max_table_files_size)}},
-        {"ttl",
-         {offset_of(&CompactionOptionsFIFO::ttl), OptionType::kUInt64T,
-          OptionVerificationType::kNormal, true,
-          offsetof(struct CompactionOptionsFIFO, ttl)}},
```

Comment:
Need to make it Deprecate. We should never remove options from the map.

Reply:
yeah, this should be deprecated.

Reply:
I was thinking we would later update all fbcode callsites to make use of ColumnFamilyOptions.ttl. But another way to not cause disruption to existing users is to set `columnfamilyoption.ttl 
 = ComfactionOptionsFifo.ttl` in column_family.cc::SanitizeOptions.

https://github.com/facebook/rocksdb/blob/1a761e6a6cfdab685eb48c1606668094147f3631/db/column_family.cc#L181

Reply:
Ignore this particular suggestion of mine: 

> But another way to not cause disruption to existing users is to set `cf_options.ttl = ComfactionOptionsFifo.ttl` in column_family.cc::SanitizeOptions.

We will just break the compatibility as we are going to RocksDB 6.0. 
We will update fbcode call sites and external users need to update their code to set `ColumnFamilyOptions.ttl`.